### DISCUSSION TITLE
Write there your questions for me

### DISCUSSION THREAD

**Discussion by nchistov** [nchistov/nchistov#1] [Q&A]
Ok.

**Comment by Peter-developer01** [comment]
questinons !== questions

**Comment by Peter-developer01** [comment]
Have you Discord?

**Comment by nchistov** [comment]
@Peter-developer01 No, I haven't got any messangers because I don't have the phone!

**Comment by Peter-developer01** [comment]
@nchistov  Whoa!! You can register Discord even on a computer. You just need email to register for Discord.

**Comment by Peter-developer01** [comment]
Dear @nchistov,
We're writing in reference to your Stack Overflow на русском account:
https://ru.stackoverflow.com/users/507516/nchistov

Be warned that users got banned most often in 2k..3k rep.

So please don't create a sockpuppet, don't do serial voting, and **please [be nice](https://ru.stackoverflow.com/help/be-nice)**.
Otherwise you can get a suspended account.

We warned you. Thanks.

- *Regards, Stack Overflow на русском moderators*

**Reply by Peter-developer01** [reply]
A little sarcasm, but it's true.

**Comment by nchistov** [comment]
@Peter-developer01 can you say this in Russian, I can't understand.

**Reply by Peter-developer01** [reply]
> @Peter-developer01 can you say this in Russian, I can't understand.

Уважаемый @nchistov,
Это сообщение касается вашей учётной записи на сайте Stack Overflow на русском:
[https://ru.stackoverflow.com/users/507516/nchistov](https://ru.stackoverflow.com/users/507516/nchistov)

Имейте в виду, что пользователи чаще всего получают бан в периоде 2k..3k репутации.

Поэтому, пожалуйста, не создавайте [марионеток](https://en.wikipedia.org/wiki/Sockpuppet_(Internet)), не голосуйте фиктивно [(?)](https://en.wikipedia.org/wiki/Sockpuppet_(Internet)) и, пожалуйста, [будьте добры](https://ru.stackoverflow.com/help/be-nice).
В противном случае вы можете получить бан.

Мы предупреждали вас. Спасибо.

- *С уважением, модераторы Stack Overflow*

**Reply by nchistov** [reply]
Спасибо.

**Reply by Peter-developer01** [reply]
Буду надеяться, что вы не нарушите эти простые правила. Удачи!

**Reply by nchistov** [reply]
Когда вы мне написали, я подумал что я сделал что-то не так. Оказывается вы просто предупреждаете...

**Reply by Peter-developer01** [reply]
Всё вы правильно делаете, просто предупреждаю, верно :)

**Reply by Peter-developer01** [reply]
Чтобы вы не попали в ту же комнату банхаммеров [(?)](https://ru.wikipedia.org/wiki/Бан_(Интернет)#Банхаммер).

**Reply by Peter-developer01** [reply]
P.S. Забыл сказать. Найдено в ответе *A K* [здесь](https://ru.meta.stackoverflow.com/questions/11288/Модерация-за-2020-год-в-цифрах/11302#11302) (читайте абзац 6).

**Comment by Peter-developer01** [comment]
all Chistov's are your brothers? :D

**Reply by nchistov** [reply]
No, only @wcistow, @FrCln is my father :D.

**Reply by Peter-developer01** [reply]
Okay understood. :D

**Comment by nchistov** [comment]
@Peter-developer01 Do you know [gitflic](https://gitflic.ru)?

**Reply by Peter-developer01** [reply]
Not so good, but yes. https://gitflic.ru/user/peter-developer01
But how is it different from GitHub, except language?

**Comment by Peter-developer01** [comment]
@nchistov 
> Самый обыкновенный Подросток N...

can I ask what's your age? You can skip this question of course, no harassment. Just wonder ;)

**Reply by nchistov** [reply]
Oh I broke the ruSO rules at first because I register there when I am 12, but now I am already 13.

**Reply by nchistov** [reply]
Do you know where I got this "Teenager N" from?

**Reply by nchistov** [reply]
This is Nina Dashevskaya's short story "Подросток _N_". https://fantlab.ru/work1254304

**Reply by Peter-developer01** [reply]
Haha, understand.

**Reply by nchistov** [reply]
@Peter-developer01 and what's your age?

**Reply by Peter-developer01** [reply]
> Oh I broke the ruSO rules at first because I register there when I am 12, but now I am already 13.

Oh, that's not "ruSO rules", that is law. You broke not just SO rules. But not anymore.

**Reply by Peter-developer01** [reply]
> @Peter-developer01а какой у тебя возраст?

I'm not going to reveal it.

**Reply by nchistov** [reply]
@Peter-developer01 yes.

**Reply by Peter-developer01** [reply]
Yes what?

**Reply by nchistov** [reply]
@Peter-developer01

> Oh, that's not "ruSO rules", that is law. You broke not just SO rules. But not anymore.

**Comment by Peter-developer01** [comment]
3... 2... 1... Prepare... *Entering a new step...*

🇨‌🇴‌🇳‌🇬‌🇷‌🇦‌🇹‌🇺‌🇱‌🇦‌🇹‌🇮‌🇴‌🇳‌🇸‌ on reaching 2k reputation! :tada:

**Reply by nchistov** [reply]
Oh, thank you. I thought no one would congratulate me when I reach 2k rap. Thank you.

**Reply by nchistov** [reply]
Oh, 5 up votes? It is you,  thank you. But serial voting is bad, learn it by heart.

**Reply by Peter-developer01** [reply]
Yes yes but it's not serial voting, its a bit different

**Comment by Peter-developer01** [comment]
I have approximately 1060 helpful flags network-wide. I am proud of that. How many helpful flags you have network-wide? ;)

**Comment by Peter-developer01** [comment]
@nchistov Sorry, I'm so busy now!

**Reply by nchistov** [reply]
@Peter-developer01 And after 5 days I won’t be able to use github. Because of 2FA. I haven't got a phone so I can't scan QR code. :(((

**Reply by Peter-developer01** [reply]
@nchistov I'll search how we can solve it!

**Reply by Peter-developer01** [reply]
@nchistov I think you don't need the phone, I'll try to enable 2FA at my own risk and see if I would login with it!

**Reply by Peter-developer01** [reply]
Dang GitHub! It's not fair.

**Reply by Peter-developer01** [reply]
And I've again experiencing an problem cascade, I can't signup for 1password, my deadlines is burning, 1password is trial, I can't use phone. Aaaaaaaaahhhhh.

**Reply by Peter-developer01** [reply]
If I won't find a solution I'll move to GitLab. But I don't want. :(

**Reply by Peter-developer01** [reply]
I DID FOUND A SOLUTION

But for Linux.

**Reply by Peter-developer01** [reply]
@nchistov Come on! Let's configure 2FA on your profile together. No need to fear - I've found an 2FA authenticator.

**Comment by Peter-developer01** [comment]
Hey @nchistov! Check out https://holopin.io!

**Reply by nchistov** [reply]
What is it?

**Reply by Peter-developer01** [reply]
It is Digital Badges. I've got an Early Bird badge.

**Comment by Peter-developer01** [comment]
@nchistov Happy new year if you celebrate it lol!

**Comment by Peter-developer01** [comment]
Hey, how are you doing?

**Reply by nchistov** [reply]
Great, thanks! 😄 But not in programming. I haven't been programming much lately (no ideas, and no time). But I started (not started, continued again) a C course. C is very interesting language. Many things I can't do as easy as in python. But I love it (not always 😅), because it trains my brain.

And how are you? What projects are you working on?

**Reply by Peter-developer01** [reply]
I am doing great, thanks.

Wow, C is sure a great language. I loved modding [NetHack](https://github.com/NetHack/NetHack) and its variants even though I knew nothing about C, and was very successful in it despite understanding little! I just looked at the existing patterns, and assembled entirely new features from small pieces. Easy.

I am currently continuing a Full-Stack web developing course and currently working on [sf-scan-project](https://github.com/Peter-developer01/sf-scan-project) (you can see how it should look like when finished in [the Figma layout](https://www.figma.com/design/u3MOjzYnTnirz712GrLbFv/Макет-СКАН?node-id=6-34204&t=XlHqhHy7mnZi6IcP-0)).  
Generally, HTML + CSS + JS, but currently React (combined HTML and JS) and CSS. I am gonna also learn some dev-ops so that I can also publish sites easily when I buy a VPS/hosting/whatever (though it's not a full-stack dev's work).

Currently this project is front-end only (HTML/CSS/JS, the thing you see), but this is the final project in the course before the back-end (mostly PHP and Node.js, the thing that makes the thing you see not be just a static image - the same thing that lets me post comments here!).

I am also trying to find some work already, and has built one landing page already for a real client ([ekors-design.ru](https://ekors-design.ru), [Peter-developer01/ekors.design](https://github.com/Peter-developer01/ekors.design)). After this, I have tried to publish an Avito ad, but got zero views for two or three weeks already. Oh well, guess I am not very appealing because I have no client history, no paid Avito "продвижения", and do not do too complex projects.

Precisely because of this, I am trying to advance my skills so I can be more appealing.

The competition is ruthless!

Been in programming for almost 5 years already, since 10.

I'm sorry if I'm too verbose, I like sharing details.

**Reply by Peter-developer01** [reply]
I just checked out on Stack Exchange (which is also trying to rename itself into Stack Overflow), it is such an unbelievably toxic place. Just wow!

**Reply by Peter-developer01** [reply]
Well, and did I just talk to a wall?

**Reply by nchistov** [reply]
No. Sorry, I forgets to answer you sometimes. And also I don't know, what to answer. I can't say which project am I working on, because there isn't any... Last time I visits StackOverflow rarely. I haven't got questions (because I program not much and have some other things to do such as summer education) and can't write good answers due to the decreased number of questions (ChatGPT killed SO).

**Reply by Peter-developer01** [reply]
Heh. Funny.
English Stack Overflow has lots of questions but even if they are good ones, I can't answer them (and I assume you can't too), because they are all extremely specific.

**Reply by nchistov** [reply]
And because there are so many people, like us, who answer faster 😄

**Reply by Peter-developer01** [reply]
Yeah, that too.
